react.js

```diff
@@ -0,0 +1,71 @@
+module.exports = {
+	extends: ["airbnb", "airbnb/hooks"],
+	plugins: [],
+	env: {
+		es6: true,
+		node: true,
```

Comment:
Do we really need `node` env for frontend? I would also add `jest: true` here for future tests.

Reply:
no we not need it, I removed it and add the `jest`, also I remove the es6 and add the `es2022`